src/main/java/core/selenium/WebDriverManager.java

```diff
@@ -72,4 +72,22 @@ private void initialize() {
         wait = new WebDriverWait(webDriver, webDriverConfig.getExplicitWaitTime(),
                 webDriverConfig.getWaitSleepTime());
     }
+
+    /**
+     * Sets the web driver waits to the minimum to handle datatable elements.
+     */
+    public void setTableWaitMode() {
+        this.webDriver.manage().timeouts().implicitlyWait(0, TimeUnit.SECONDS);
+        wait = new WebDriverWait(webDriver, 0, 0);
+    }
+
```

Comment:
Both code are the same, maybe create a new method or something like that

Reply:
Solved